mobile/joseki/joseki.xcodeproj/project.pbxproj

```diff
@@ -0,0 +1,807 @@
+// !$*UTF8*$!
```

Comment:
this is what goes on underneath the xcodeproj file which contains information on how to compile